services/portal/templates/deployment.yaml

```diff
@@ -44,6 +44,28 @@ spec:
               value: {{ include "portal.fullname" . }}-redis
             - name: "PROPS_sso__req__auth__hosts"
               value: {{ .Values.global.host | quote }}
+            - name: "PROPS_lsst__hips__masterUrl"
+              value: https://irsa.ipac.caltech.edu/data/hips/list
+            - name: "PROPS_FIREFLY_OPTIONS"
+              value: >-
+                $'{
+                   "coverage":  {"hipsSourceURL" : "ivo://CDS/P/2MASS/color"},
```

Comment:
This needs to be the Rubin `color_gri` now, right?

Reply:
Yes, but I could not test it because I don't think it is available on dev

Reply:
also we will have add more to the values table.